docs/security-configuration/proxy.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: default
+title: Proxy-based authentication
+parent: Security - Configuration
+nav_order: 20
+---
+
+# Proxy-based authentication
+
+If you already have a single sign-on (SSO) solution in place, you might want to use it as an authentication backend.
+
+Most solutions work as a proxy in front of Elasticsearch and the Security plugin. If proxy authentication succeeds, the proxy adds the (verified) username and its (verified) roles in HTTP header fields. The names of these fields depend on the SSO solution you have in place.
+
+The Security plugin then extracts these HTTP header fields from the request and uses the values to determine the user's permissions.
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## Enable proxy detection
+
+To enable proxy detection for Elasticsearch, configure it in the `xff` section of `config.yml`:
+
+```yml
+---
+_meta:
+  type: "config"
+  config_version: 2
+
+config:
+  dynamic:
+    http:
+      anonymous_auth_enabled: false
+      xff:
+        enabled: true
+        internalProxies: '192\.168\.0\.10|192\.168\.0\.11'
+        remoteIpHeader: 'x-forwarded-for'
+```
+
+You can configure the following settings:
+
+Name | Description
+: ---| :---
```

Comment:
I think there's an extra space in ": ---"

Reply:
Thanks a bunch, resolved.